README.md

```diff
@@ -60,12 +60,14 @@ The SEO tag will respect any of the following if included in your site's `_confi
     username: benbalter
   ```
 
-* `facebook:app_id` (a Facebook app ID for Facebook insights), and/or `facebook:publisher` (a Facebook page URL or ID of the publishing entity). You'll want to describe one or both like so:
+* `facebook:app_id` (a Facebook app ID for Facebook insights), `facebook:publisher` (a Facebook page URL or ID of the publishing entity) and/or `facebook:admins`
```

Comment:
To improve readability, I'd break this into a short intro and three bullets like we do for `social`.